ui/src/shared/components/LayoutCellMenu.js

```diff
@@ -63,31 +86,17 @@ class LayoutCellMenu extends Component {
               {queries.length ? (
                 <MenuTooltipButton
                   icon="pencil"
-                  menuOptions={[
-                    {text: 'Configure', action: onEdit(cell)},
-                    {
-                      text: 'Add Annotation',
-                      action: onStartAddingAnnotation,
-                      disabled: !cellSupportsAnnotations(cell.type),
-                    },
-                    {
-                      text: 'Edit Annotations',
-                      action: onStartEditingAnnotation,
-                      disabled: !cellSupportsAnnotations(cell.type),
-                    },
-                  ]}
+                  menuOptions={menuOptions}
                   informParent={this.handleToggleSubMenu}
                 />
               ) : null}
-              {dataExists && (
```

Comment:
any risk of losing this check on `dataExists`? what was it for?

Reply:
Previously it would only show the download button if the cell had data available. I have it now so it always shows the download option but disables it when there is no data to download